test/server/cards/01_SOR/units/CargoJuggernaut.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Cargo Juggernaut', function () {
+    integration(function () {
+        describe('Cargo Juggernaut\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['cargo-juggernaut'],
+                        groundArena: ['pyke-sentinel'],
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors']
+                    }
+                });
+            });
+
+            it('should heal base', function () {
```

Comment:
Could we be a little clearer for someone who might be reading the test without knowing the card: 'should heal base when there is a friendly Vigilance unit'. Same for the other test